test/inductor/test_select_algorithm.py

```diff
@@ -109,7 +109,6 @@ def foo(a, b):
         )
         self.assertEqual(counters["inductor"]["select_algorithm_autotune"], 1)
 
-    # FIXME: Investigate why _int_mm_out_cuda is not compiled on ROCm
```

Comment:
Why this is removed.

Reply:
This changes was made when resolving the conflict with commit https://github.com/ROCm/pytorch/commit/9521966dfed08c968507888a26f0c502a576d9a9